tools/docker/Dockerfile.integ-test

```diff
@@ -53,6 +53,7 @@ RUN --mount=type=bind,target=/src make -C /src \
   install-test-cni-bins \
   demo-network
 
+COPY tools/image-builder/rootfs-slow-reboot.img /var/lib/firecracker-containerd/runtime/rootfs-slow-reboot.img
```

Comment:
Maybe we should have something like a `install test-depends` target and include installing this along with the pre-existing `install-test-cni-bins` target. If we are going to want to support testing outside containers directly on bare-metal hosts (like we were talking about before), we'd get a lot of benefit from being able to just run the same `install test-depends` target on both the host and inside a container rather than duplicate stuff like this copy between them.

Otherwise LGTM

Reply:
Sounds good. Will do.